### DISCUSSION TITLE
Change InplaceAdd so that whatever is in the Search field automatically populates a field in form opened upon click of "No results"

### DISCUSSION THREAD

**Discussion by DGoertzen** [serenity-is/Serenity#6742] [Serenity Ideas]
I think it would be more user friendly if, when using InPlaceAdd, if a user types something in the quicksearch, and then clicks "***No results.  Click to define a new one***", then one field on the form that is opened populates with what was in the quicksearch box.

For example, in the demo at   https://demo.serenity.is/Meeting/Meeting#edit/1, if I enter "Employee Initiation" into the Meeting Type as shown:
![image](https://user-images.githubusercontent.com/22012694/229338002-b4dba178-0a12-4be2-b4f8-6850bb16e353.png)

and then click "***No results.  Click to define a new one***", I'd like the "New Meeting Type" to open with this:

![image](https://user-images.githubusercontent.com/22012694/229338045-28fac647-ac6f-4205-8b80-b6cb6f985291.png)

Also, if there is only one field to be completed, like in the above example, I'd like tbe option to be able to just have that item automatically created after clicking ""***No results.  Click to define a new one***" without having a "New Meeting Type" form open at all.

That is how I plan to make mine behave, and I thought others in the community might also like this behavior as it saves the user from retyping.  If this is already implemented somehow, please let me know.

**Comment by choss21** [comment]
En versiones anteriores asi funcionaba, nose porque lo quitaron, lo de rellenar automaticamente la opcion a agregar

**Comment by volkanceylan** [comment]
In 8.5.2 this should work for setting name property. But currently no to "without having to open a form at all" part, it is not possible.
